acme/acme.go

```diff
@@ -333,7 +334,8 @@ func (a *ACME) CreateLocalConfig(tlsConfig *tls.Config, certs *safe.Safe, checkO
 
 	a.client, err = a.buildACMEClient(account)
 	if err != nil {
-		return err
+		log.Errorf("Failed to build ACME Client: %s", err)
```

Comment:
Could we make this message more verbose?
"Failed to build ACME client" doesn't really tell a whole lot. Would make this message more like:
"Failed to build ACME client. LetsEncrypt functionality will be limited until traefik is restarted."